pkgs/development/python-modules/pymorphy3/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, isPy3k
+, dawg-python
+, docopt
+, pymorphy3-dicts-ru
+, pymorphy3-dicts-uk
+}:
+
+buildPythonPackage rec {
+  pname = "pymorphy3";
+  version = "1.2.0";
+
+  disabled = !isPy3k;
```

Comment:
We no longer support python2. Please use pythonOlder/pythonAtLeast and a version number

Reply:
Good catch. Looks like all versions of python3 are supported, so I removed `disabled` altogether.